docs/packaging/creating-a-new-package.md

```diff
@@ -83,11 +118,13 @@ install    : |
     install -Dm00755 $workdir/tree $installdir/usr/bin/tree
 ```
 
-Understanding how to translate source code into a good `package.yml` file is the heart of packaging. If you are stumped, or have questions, **ask for help on [Matrix](/docs/user/contributing/getting-involved#matrix-chat).**
+To read more about finding and including dependencies and other parts of `package.yml` please see [Packaging Practices](packaging-practices.md).
+
+Understanding how to translate source code into a good `package.yml` file is the heart of packaging. If you are stumped, or have questions, **ask for help in our Solus Packaging room on [Matrix](/docs/user/contributing/getting-involved#matrix-chat).**
 
 ## Build the Package
 
-Build the package:
+Build the package (`tree` uses `make`, other packages may use another command):
```

Comment:
What build system the software being packaged uses is irrelevant; to build a package it will always be `make` (until we switch to `go-task`).

Reply:
Unless you mean like `make local`? Either way, it might give readers the impression that the package's build system matters.

Reply:
Ah, thanks for clarifying that. I had conflated the invocation in package.yml. with the command line build command. Fixor.